scripts/qiita

```diff
@@ -130,24 +130,31 @@ def load_study(owner, title, info):
               help="The parent artifacts from which the new artifact has been "
                    "processed",
               type=click.IntRange(1))
-@click.option('--command', required=False,
-              help="The command used to process the new artifact",
-              type=click.IntRange(1))
-@click.option('--parameter_set', required=False,
-              help="The parameter set used to process the new artifact",
+@click.option('--dflt_params', required=False,
```

Comment:
The benefit is that you don't need to have 2 windows open, one to check the commands/params (ipython or DB) and other one to write your command. Again, a nice to have.